services/apps/alcs/src/portal/inbox/notice-of-intent/inbox-notice-of-intent-view.entity.ts

```diff
@@ -38,7 +38,11 @@ import { LinkedStatusType } from '../inbox.dto';
         'status',
       )
       .from(NoticeOfIntentSubmission, 'noi_sub')
-      .innerJoin(NoticeOfIntent, 'noi', 'noi.file_number = noi_sub.file_number')
+      .innerJoin(
+        NoticeOfIntent,
+        'noi',
+        'noi.file_number = noi_sub.file_number AND noi.hide_from_portal = false',
```

Comment:
Nitpick: I liked the gigantic `FALSE` you had in application, more readable

Reply:
Going to ignore this as i don't want to regenerate and update the migrations for a nitpick.